code/game/objects/items/plushes.dm

```diff
@@ -542,3 +542,133 @@
+	icon_state = "realgoat"
+	squeak_override = list('sound/items/goatsound.ogg'=1)
+
+/obj/item/toy/plush/goatplushie/angry/kinggoat
+	name = "King Goat Plushie"
+	desc = "A plushie depicting the king of all goats."
+	icon_state = "kinggoat"
+	cooldown_modifier = 3
+	throwforce = 25
+	force = 25
+	attack_verb = list("chomped")
+
+/obj/item/toy/plush/goatplushie/angry/ascendedkinggoat
+	name = "Ascended King Goat Plushie"
+	desc = "A plushie depicting the god of all goats."
+	icon_state = "ascendedkinggoat"
+	cooldown_modifier = 1
+	throwforce = 30
+	force = 30
+	attack_verb = list("chomped")
+
+/obj/item/toy/plush/goatplushie/angry/ascendedkinggoat/attackby(obj/item/I,mob/living/user,params)
+	if(I.get_sharpness())
+		user.visible_message("<span class='notice'>[user] attempts to stab [src]!</span>", "<span class='suicide'>The knife bounces off [src]'s back before breaking into millions of pieces... [src] glares at [user]!</span>") // You fucked up now son
+		I.play_tool_sound(src)
+		qdel(I)
+		user.gib()
+
+/obj/item/toy/plush/goatplushie/angry/kinggoat/attackby(obj/item/I,mob/living/user,params)
+	if(I.get_sharpness())
+		user.visible_message("<span class='notice'>[user] stabs [src] to shreds!</span>", "<span class='notice'>[src]'s death has attracted the attention of the king goat plushie guards!</span>")
+		I.play_tool_sound(src)
+		qdel(src)
+		var/turf/location = get_turf(user)
+		new/obj/item/toy/plush/goatplushie/angry/masterguardgoat(location)
+		new/obj/item/toy/plush/goatplushie/angry/masterguardgoat(location)
+		new/obj/item/toy/plush/goatplushie/angry/masterguardgoat(location)
+		new/obj/item/toy/plush/goatplushie/angry/masterguardgoat(location)
+		new/obj/item/toy/plush/goatplushie/angry/guardgoat(location)
+		new/obj/item/toy/plush/goatplushie/angry/guardgoat(location)
+		new/obj/item/toy/plush/goatplushie/angry/guardgoat(location)
+		new/obj/item/toy/plush/goatplushie/angry/guardgoat(location)
+		new/obj/item/toy/plush/goatplushie/angry/guardgoat(location)
+		new/obj/item/toy/plush/goatplushie/angry/guardgoat(location)
+		new/obj/item/toy/plush/goatplushie/angry/guardgoat(location)
+		new/obj/item/toy/plush/goatplushie/angry/guardgoat(location)
+
+/obj/item/toy/plush/goatplushie/angry/kinggoat/attackby(/obj/item/reagent_containers/food/snacks/grown/cabbage/C,mob/living/user,params)
+		var/C = /obj/item/reagent_containers/food/snacks/grown/cabbage
+		user.visible_message("<span class='notice'>[user] watches as [src] takes a bite out of the cabbage!</span>", "<span class='notice'>[src]'s fur now starts glowing it seems it has ascended!</span>")
+		playsound(src, 'sound/items/eatfood.ogg', 50, 1)
+		qdel(C)
+		qdel(src)
+		var/turf/location = get_turf(user)
+		new/obj/item/toy/plush/goatplushie/angry/ascendedkinggoat(location)
+
+
+/obj/item/toy/plush/goatplushie/angry/guardgoat
+	name = "guard goat plushie"
+	desc = "A plushie depicting one of the king goats guards, tasked to protecting the king at all costs."
```

Comment:
*goat's

also, should "king goat" be capitalized? is it a proper name?

also, change "protecting" to "protect"

Reply:
I'm gonna stop commenting about grammar on this PR from here on out; please proofread your descriptions and names.

Reply:
don't forget to add punctuation marks to the ends of your descriptions that don't end with them already.